essence-of-live-coding/test/Cell.hs

```diff
@@ -23,10 +25,45 @@ import LiveCoding
 
 import qualified Cell.Util
 import qualified Cell.Monad.Trans
+import Control.Exception (ErrorCall)
+import Test.Framework.Providers.HUnit
+import Control.Exception.Base
+import Test.HUnit
 
 test = testGroup "Cell"
   [ testProperty "steps produces outputs"
     $ \(inputs :: [Int]) -> inputs === fst (runIdentity $ steps (id :: Cell Identity Int Int) inputs)
+  , testProperty "traverse' works as expected for Lists"
+    $ \(inputs :: [[Int]]) -> 
+      fmap (fmap (+10)) inputs === fst (runIdentity $ steps (traverse' (arr (+10) :: Cell Identity Int Int)) inputs)
+  , testProperty "traverse' works as expected for Maybes"
+    $ \(inputs :: [Maybe Int]) -> 
+      fmap (fmap (+10)) inputs === fst (runIdentity $ steps (traverse' (arr (+10) :: Cell Identity Int Int)) inputs)
```

Comment:
Are these tests duplicate to the ones in `test/Cell/Util.hs`?

I think all tests regarding `traverse'` should be in `test/Cell/Util.hs`, and ideally in a separate `testGroup "traverse'"`.

(I know not all tests are sorted like that currently, but they should be ideally.)

Reply:
Ah, yes, has forgotten to delete them in Util.hs, will move tests to Util.hs